### ISSUE TITLE
feature request: Per event downsampling 

### ISSUE THREAD

**Issue by liyiy** [PostHog/downsampling-plugin#8]
Allow for specific events to be downsampled

ex: Certain events are less important than others and it would be nice to be able to downsample specific events

**Comment by jordn** [comment]
yeah strong +1. Events follow a powerlaw. Would want to make sure we capture the rare events. can downsample on the big ones.
